vimeo-networking/src/main/java/com/vimeo/networking/utils/VimeoNetworkUtil.java

```diff
@@ -85,8 +84,6 @@ public static Gson getGson() {
     public static GsonBuilder getGsonBuilder() {
         // Example date: "2015-05-21T14:24:03+00:00"
         return new GsonBuilder().registerTypeAdapterFactory(new Stag.Factory())
-                .registerTypeAdapterFactory(new PrivacyDeserializer.TypeAdapterFactory())
-                .registerTypeAdapterFactory(new VideoFileDeserializer.TypeAdapterFactory())
```

Comment:
I can't remember for the life of me. There's synchronized access on the call objects so maybe it was an optimization around if we had a really long list of calls to cancel...?